eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Rewards.hs

```diff
@@ -813,11 +814,11 @@ tests =
     "Reward Tests"
     [ testProperty "Sum of rewards is bounded by reward pot" $
         withMaxSuccess numberOfTests (rewardsBoundedByPot (Proxy @C))
-    , testProperty "compare with reference impl, no provenance, v3" $
+    , testProperty "compare with reference impl, no provenance, v3" . noShrinking $
         newEpochProp chainlen (oldEqualsNew @C (ProtVer (natVersion @3) 0))
-    , testProperty "compare with reference impl, no provenance, v7" $
+    , testProperty "compare with reference impl, no provenance, v7" . noShrinking $
         newEpochProp chainlen (oldEqualsNew @C (ProtVer (natVersion @7) 0))
-    , testProperty "compare with reference impl, with provenance" $
+    , testProperty "compare with reference impl, with provenance" . noShrinking $
```

Comment:
Do you still the noShrinking? Is shrinking actually problems?

Reply:
yeah, I think the shrinkers are broken, I should probably raise an issue about this